ocf_datapipes/load/nwp/nwp.py

```diff
@@ -124,3 +132,20 @@ def check_if_physical_limits(self, nwp: Union[xr.DataArray, xr.Dataset]):
                         raise ValueError(
                             f"NWP data {var_name} is outside physical limits: ({lower},{upper})"
                         )
+    def check_if_nans(self, nwp: Union[xr.DataArray, xr.Dataset]):
```

Comment:
@peterdudfield does this look right? [ref](https://github.com/openclimatefix/ocf_datapipes/pull/340#discussion_r1689838186)

Reply:
yea, this looks good

Reply:
alrighty, lemme know if you are ready to merge